virtualize.sh

```diff
@@ -148,6 +148,12 @@ ssh $SSHOPTS root@$installserverip "
 
 kill ${tail_job}
 
+# Dump elasticsearch logs into /var/log/elasticdump,
+# upload_logs will update the dump in swift.
+scp $SSHOPTS $ORIG/dumpelastic.py root@$installserverip:/tmp
```

Comment:
Well, the thing is that ./dumpelastic.py pull the document from the database and then dispatch each log message in its corresponding file.

If we do that, we will pull the log from the installserver and then push them in the install server's FS which is not efficient at all :/.